nominees/project-simoy.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Project Simoy",
+  "aliases": [
+    ""
+  ],
+  "description": "This is a collection of data visualization projects that gives a “breezy” experience to all data enthusiasts.",
+  "website": "https://projectsimoy.com/",
+  "license": [
+    {
+      "spdx": "CC-BY-NC-4.0",
+      "licenseURL": "https://wordpress.com/home/projectsimoy.com"
```

Comment:
The `licenseURL` redirects to a private page which requires login access. @lacabra and @conradsp  I will revert once I have heard from the person who submitted Project Simoy.